kodilla-cucumber/src/main/java/seasonchecker/SeasonChecker.java

```diff
@@ -0,0 +1,12 @@
+package seasonchecker;
+
+import java.time.MonthDay;
+
+public class SeasonChecker {
+    public String checkIfSummer(MonthDay date) {
```

Comment:
co to?

Reply:
a to pośpiech w chęci zrobienia od razu zadania :( przepraszam, już się poprawiłam :)